### DISCUSSION TITLE
Notify end user that "connection has been closed"

### DISCUSSION THREAD

**Discussion by xBr0th3rx** [rustdesk/rustdesk#9334] [Feature Request]
Would it be possible to build into the Rustdesk client or for the custom client generator that the user receives a notification that the connection to him has been closed?

And then add a timer so that the notification window closes itself again.

**Comment by conradRz** [comment]
bump - I have the same request. 

Anydesk also does not have this, so Rustdesk would be likely first to have it.

I connect to the remote, as the remote informs me via sound if something at my home goes wrong. But since there is no notification when connection to the remote drops on the client, I miss stuff.
